dev-packages/e2e-tests/test-applications/solidstart/tests/performance.client.test.ts

```diff
@@ -0,0 +1,93 @@
+import { expect, test } from '@playwright/test';
+import { waitForTransaction } from '@sentry-internal/test-utils';
+
+test('sends a pageload transaction', async ({ page }) => {
+  const transactionPromise = waitForTransaction('solidstart', async transactionEvent => {
+    return !!transactionEvent?.transaction && transactionEvent.contexts?.trace?.op === 'pageload';
```

Comment:
You probably want to assert for a particular transaction name, otherwise the tests are gonna start leaking once you add more page.goto()s

Reply:
Updated